cmake-ide.el

```diff
@@ -441,6 +441,9 @@ the object file's name just above."
 (defun cmake-ide--get-build-dir ()
   "Return the directory name to run CMake in."
   (when (not cmake-ide-dir) (setq cmake-ide-dir (make-temp-file "cmake" t)))
+  (when (not (or (string-prefix-p "/" cmake-ide-dir)
```

Comment:
Instead of `string-prefix-p` it should be `file-name-absolute-p`. As it is now it'd never work on Windows.